druid/src/theme.rs

```diff
@@ -42,6 +42,12 @@ pub const BORDERED_WIDGET_HEIGHT: Key<f64> = Key::new("bordered_widget_height");
 
 pub const SCROLL_BAR_COLOR: Key<Color> = Key::new("scroll_bar_color");
 pub const SCROLL_BAR_BORDER_COLOR: Key<Color> = Key::new("scroll_bar_border_color");
+pub const SCROLL_BAR_MAX_OPACITY: Key<f64> = Key::new("scroll_bar_max_opacity");
+pub const SCROLL_BAR_FADE_DELAY: Key<u64> = Key::new("scroll_bar_fade_time");
```

Comment:
these first two items are borderline, to me, in terms of whether they should be in a theme. This basically touches on some very foundational ideas about what a theme is supposed to be and what it's supposed to do. Do we want to be tuning these things per-platform, or to generally let the user tune them? It's possible that we do, but if we really want to be 'native' feeling per-platform it's likely going to take more than this.

Anyway, I'm rambling. I don't think it's bad for these to be here, per-se, I just think it's not quite clear exactly what the scope of a theme is, and this is something we're going to have to figure out. 👌

Reply:
I placed these here as someone who has plans to use Druid in an application I know that these are values I want control over without modifying Druid itself as a dep for said application